source/manual/alerts/publisher-unprocessed-fact-check-emails.html.md

```diff
@@ -1,63 +1,83 @@
+
+#### Retrieving credentials
 
-The password to log into the govuk-fact-check@digital.cabinet-office.gov.uk
-inbox can be found by running:
+The passwords for the @digital.cabinet-office.gov.uk addresses are found in
+[govuk-secrets]:
 
-```bash
+[govuk-secrets]: https://github.com/alphagov/govuk-secrets
+
+```sh
 PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline pass google-accounts/govuk-fact-check@digital.cabinet-office.gov.uk
 ```
 
-The password to log into the factcheck@alphagov.co.uk inbox can be found by
-logging into the Publisher console:
+The password for the factcheck@alphagov.co.uk address can be found by logging
+into the Publisher console:
 
-```bash
-gds govuk connect app-console -e production publisher
+```sh
+$ gds govuk connect app-console -e production publisher
 ```
 
-and then running:
-
-```bash
-Publisher::Application.mail_fetcher_config
+```ruby
+> Publisher::Application.mail_fetcher_config
 ```
 
-The same can be done for
-staging (govuk-fact-check-staging@digital.cabinet-office.gov.uk)
-and integration (govuk-fact-check-integration@digital.cabinet-office.gov.uk).
+### Investigate the unprocessed emails
 
-[fact-check]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/doc/fact-checking.md
-[script]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/script/mail_fetcher
-[schedule]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/config/schedule.rb#L7-L9
-[process]: https://github.com/alphagov/publisher/blob/d0ab32c10c5d22ffb9b6edccb84f5345bd766cf4/lib/fact_check_email_handler.rb#L41-L53
-[warning]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp#L19
-[critical]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp#L20
-[login]: https://support.google.com/accounts/answer/1721977?co=GENIE.Platform%3DDesktop&hl=en
-[alert]: https://github.com/alphagov/govuk-puppet/blob/68507dba280dd6e2fd4e5663f915c0fd7be06bef/modules/govuk/manifests/apps/publisher/unprocessed_emails_count_check.pp
+There are a number of different ways of proceeding here based on why the email
+wasn't able to be processed:
+
+#### Spam/out of office
+
+If the email is clearly spam and not relevant, **it can be deleted**.
+
+#### Missing edition identification
+
+If the subject line is missing the edition identification but is clearly a
+valid fact check response, the first thing to check is if a follow up
+email was sent with the correct subject.
+
+This can be checked by [logging in to Publisher][publisher] and finding the
+relevant edition by using the title. Usually you'll need to filter on editions
+with the state of "Published" or "Out for fact check".
+
+Once you've found the edition, you can check in the "History" tab to see if
```

Comment:
```suggestion
Once you've found the edition, you can check in the "History and notes" tab to see if
```

Reply:
Good point, I was writing this from memory!